gatorgrade/output/output_functions.py

```diff
@@ -98,3 +98,25 @@ def output_failed_checks(failed_checks):
         print(f"{color.Fore.RED}\u2718  {color.Style.RESET_ALL}{requirement}")
         print(f"    {color.Fore.YELLOW}\u2192  {description}")
         return bool
+
+
+def run_and_display_command_checks(commands):
+    """Run commands through gatorgrader and display them to the user.
+
+    Args:
+        Commands are received as dictionary of two keys, shell commands / gator
+        commands.
+
+        commands_input (dict{str: List[dict{str:str, ...}],
+        str: List[List[str]]}): The first parameter which
+        contains commands.
+
+        An Example of arg input for this function is as follows :
+
+        {'shell': [{'description': 'Run program', 'command': 'mdl'}],
+        'gatorgrader': [['--description', 'do command', 'commandType',
+        '--arg', '1', '--directory', './home', '--file', 'file.py']]}
+    """
+    results = run_commands_and_return_results(commands)
+    sort_checks_by_result(results)

```

Comment:
In looking at this function now (`sort_checks_by_result`), it does a lot more than just sorting the checks -- perhaps it can be renamed to something like `display_check_results`?

Reply:
I hesitate to make that change because that function is from @lukel43's PR that already merged with Main.  We can go back through and refactor but I'd want to make sure he's aware and able to change the other things that would affect.

Reply:
Upon further inspection it looks like that function isn't called anywhere else so it looks like changing it won't be a big deal.  I made the change locally and I'll go ahead and commit them now.